docs/User_Guide.md

```diff
@@ -329,10 +329,84 @@ The following table summarizes how the final bind path is computed:
 | /home/foo       | non-existent         | /home/foo                            |
 | /home/foo       | /some/path/root      | /some/path/root/ServiceBinding_Name  |
 
+## Custom naming strategy
+Naming strategy defines a way to prepare binding names through
+ServiceBinding request.
```

Comment:
we need to state here the motivation, i.e. the problem we solve with this.

Reply:
done